config/sidekiq.yml

```diff
@@ -7,4 +7,6 @@ production:
   - default
   - user_verifications
   - mailers
+  - active_storage_analysis
```

Comment:
these queues are not necessary for us

Reply:
Agh, you're right, it comes from a test I made and I missed to delete it